src/devices/bresser_7in1.c

```diff
@@ -158,11 +226,9 @@ static int bresser_7in1_decode(r_device *decoder, bitbuffer_t *bitbuffer)
         int pm_2_5 = (msg[10] & 0x0f) * 1000 + (msg[11] >> 4) * 100 + (msg[11] & 0x0f) * 10 + (msg[12] >> 4);
         int pm_10  = (msg[12] & 0x0f) * 1000 + (msg[13] >> 4) * 100 + (msg[13] & 0x0f) * 10 + (msg[14] >> 4);
 
-        // To Do: identify further data
-
         /* clang-format off */
         data = data_make(
-                "model",            "",                         DATA_STRING, "Bresser-7in1",
+                "model",            "",                         DATA_STRING, "Bresser-AIR-PM",
```

Comment:
You can add `// should be "Bresser-Air"` but you can not change existing model names.